src/errors/DatabaseError.js

```diff
@@ -0,0 +1,26 @@
+export default class DatabaseError extends Error {
+    constructor(message, options={}) {
+      super(message)
+      this.name = 'DatabaseErr'
+      this.message = 'Data-related error'
+      this.statusCode = 500
+      this.err = null
+  
+      if(options) {
+        for (const [key, value] of Object.entries(options)) {
+          this[key] = value
+        }
+      }
+  /*
+      if(this.err && this.err.name === 'MongoError') {
```

Comment:
Would it be possible to clear this out if it is commented out

Reply:
ok